Find a functionally equivalent code:android.widget.DatePicker.setSpinnersShown:COMMENT
Method Modifier: public      
Comment:/**
 * Sets whether the spinners are shown.
 *
 * @param shown True if the spinners are to be shown.
 */

@@ -1,3 +1,3 @@
 {
-    mSpinners.setVisibility(shown ? VISIBLE : GONE);
+    mDelegate.setSpinnersShown(shown);
 }

